tests/cupy_tests/core_tests/test_ndarray.py

```diff
@@ -63,6 +63,13 @@ def test_order(self):
         self.assertTrue(a.flags.f_contiguous)
         self.assertTrue(not a.flags.c_contiguous)
 
+    def test_order_None(self):
```

Comment:
Please use lower case `none` for a method name.

Reply:
OK